bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/progress/ProgressInfoItem.java

```diff
@@ -1014,7 +1016,7 @@ private boolean getCustomThemeFlag() {
 		if (engine != null) {
 			ITheme activeTheme = engine.getActiveTheme();
 			if (activeTheme != null) {
-				return !DEFAULT_THEME.equals(activeTheme.getId());
+				return DARK_THEME.equals(activeTheme.getId());
```

Comment:
the getCustomThemeFlag() says in the javadoc that it is returning true if a theme is selected. But the method returns true if the current theme is the dark one. even if this is a private method, it could be renamed into 'isDarkThemeSelected'